### PR TITLE
Add SF Symbols.app 1.0

### PR DESCRIPTION
**Pull request by jodelamo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/sf-symbols.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'f822046b1880377e4c8d1df8fd807d9335404a21245bb8abcb3481c6dd47c5a0'
+
+  url 'https://developer.apple.com/design/downloads/SF-Symbols.dmg'
+  name 'SF Symbols'
+  homepage 'https://developer.apple.com/design/human-interface-guidelines/sf-symbols/overview/'
+
+  pkg 'SFSymbols.pkg'
+
+  uninstall pkgutil: 'com.apple.SFSymbols.plist',
+            delete:  '/Applications/SF Symbols.app'
```
Comment (vitorgalvao) [inline Casks/sf-symbols.rb]: Are you sure this is necessary? Doesn’t the `pkgutil:` get rid of it?

Comment (jlowgren) [reply Casks/sf-symbols.rb]: Travis complained the first time. But I might have misinterpreted it. I can try changing it again.

Comment (jlowgren) [reply Casks/sf-symbols.rb]: Removing `:delete` will leave the app bundle in the `/Applications` directory. It's possible I'm passing the wrong file to `pkgutil:`? I'm not 100% sure what the correct file should be.

Comment (vitorgalvao) [reply Casks/sf-symbols.rb]: > It's possible I'm passing the wrong file to `pkgutil:`?

It’s likely you’re doing it right. This sometimes happens.

**Pull request merged**